R/readQFeatures.R

```diff
@@ -11,12 +11,19 @@
 ##'
 ##' - The single-set case will generate a `QFeatures` object with a
 ##'   single `SummarizedExperiment` set containing all features of the
-##'   input table.
+##'   input table. This single-set case should be used when reading 
```

Comment:
I wouldn't be so directive. Instead of *should be used*, I would suggest *are generally used*.